plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginCommandHandler.java

```diff
@@ -22,18 +25,24 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
     Credential credential = loginService.getCachedActiveCredential();
     if (credential == null) {
       credential = loginService.getActiveCredential();
+
+      if (credential != null) {
+        try {
+          new GoogleLoginTemporaryTester().testLogin(credential);
```

Comment:
Now that we have deployment, we should be getting rid of this temporary tester class.